spec/features/reviews/new_spec.rb

```diff
@@ -1,10 +1,41 @@
-# As a visitor,
-# When I visit an item's show page
-# I see a link to add a new review for this item.
-# When I click on this link, I am taken to a new review path
-# On this new page, I see a form where I must enter:
-# - a review title
-# - a numeric rating that can only be a number from 1 to 5
-# - some text for the review itself
-# When the form is submitted, I should return to that item's
-# show page and I should see my review text.
+describe "Create new review" do
+  describe "when i visit the item show page" do
+    before :each do
+      @bike_shop = Merchant.create(
+        name: "Brian's Bike Shop",
+        address: '123 Bike Rd.',
+        city: 'Denver',
+        state: 'CO',
+        zip: 80203
+      )
+      @chain = @bike_shop.items.create(
+        name: "Chain", description: "It'll never break!",
+        price: 50,
+        image: "https://www.rei.com/media/b61d1379-ec0e-4760-9247-57ef971af0ad?size=784x588",
+        inventory: 5
+      )
+    end
+    it "i can create a new review by filling out a form" do
+
+      title = "review"
+      content = "theis product is great"
+      rating = 4
+
+      visit "items/#{@chain.id}"
```

Comment:
I like to format these tests to have spaces between things like 'visit', 'expect' and 'fill in'. Should help to understand the workflow better